web/web/notifications/static/helpers.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { queue } from "../../static/js/components/app_notification_queue";
+
+export function validateEmail(email) {
```

Comment:
I think this could be dried up by using the createPopup function in helpers instead of writing queue.notify again (also capitalization in title is personal preference).